src/components/fields/ImageUploadsImage.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { toastr } from 'react-redux-toastr';
+
+import { ToastrOptionsConfirm } from '../../utilities/toastr.js';
+import { getResizedImageUrl } from '../../utilities/images.js';
+
+class ImageUploadsImage extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.imageUrl = undefined;
+
+    if(this.props.image !== undefined) {
+      this.imageUrl = getResizedImageUrl(this.props.image, '350', false);
```

Comment:
I don't think we should be using this here because when the preview is created, the 350 thumbnail doesn't exist yet.  So it never loads until you refresh the page and only if the thumbnail exists.

Reply:
yes good point (that sucks)